src/main.cpp

```diff
@@ -281,9 +281,10 @@ int main(int argc, char * argv[])
         SetConsoleTitleA("Mixxx Debug Messages");
     }
 #endif
+    UINT oldCodePage;
```

Comment:
what if GetConsoleWindow() is null?  the call below will have an uninitialized value